lib/Common/Memory/CustomHeap.h

```diff
@@ -89,7 +89,8 @@ struct Allocation
         } largeObjectAllocation;
     };
 
-    __field_bcount(size) char* address;
```

Comment:
Agree with @dilijev

I was trying to build other code today and happened to run into this too. I tried different solutions and chose to add the last line below to ```CommonPal.h```. You can consider this approach.

```
#define __in    _SAL1_Source_(__in, (), _In_)
#define __out   _SAL1_Source_(__out, (), _Out_)
#define __notnull
```